Userland/Applications/TodoList/main.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2021, Arthur Brainville <ybalrid@ybalrid.info>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/LockFile.h>
+#include <LibGUI/Application.h>
+#include <LibGUI/Menu.h>
+#include <LibGUI/Window.h>
+#include <LibGUI/WindowManagerServerConnection.h>
+#include <WindowServer/WindowManager.h>
+
+#include "TodoList.h"
+
+int main(int argc, char** argv)
+{
+    const auto user_file_path = String::formatted("{}/.todolist.json", getenv("HOME"));
+    const auto personalLockPath = AK::String::formatted("{}/.todolist.lock", getenv("HOME"));
+    Core::LockFile lockfile(personalLockPath.characters());
```

Comment:
```suggestion
    Core::LockFile lockfile("/tmp/lock/todolist.lock");
```

Reply:
I do want to keep the lock file as a "per user" thing, so if I use the /tmp/lock dir, I'm gonna add the UID of the user in the lockfile name 🙂

Reply:
`/tmp/lock/todolist-<UID>.lock` is now used